### ISSUE TITLE
Variable/Argument names in Godot's autocomplete.

### ISSUE THREAD

**Issue by StaryMic** [2shady4u/godot-sqlite#163]
**Is your feature request related to a problem? Please describe.**
I'm not sure if this is a bug or something that hasn't been added, but currently I don't have any argument names in Godot 4.2.1 Stable. I've been using the demo project as documentation since the Readme is a bit hard to read, but this is proving to be an annoying issue to deal with.

**Describe the solution you'd like**
Names for arguments please 🙏
This plugin works very well from what I've toyed with but as someone learning SQLite and databases, I need variable/argument names to help me know what data goes in each argument.

![image](https://github.com/2shady4u/godot-sqlite/assets/37278662/d23f1823-aaf8-4e35-81c6-61fc9dbcbd12)

Thank you in advance!

**Comment by 2shady4u** [comment]
Hello @StaryMic 😄 

I am limited by what is supported by the GDExtension framework. I'll check if this behaviour is implemented or not.

**Comment by 2shady4u** [comment]
@StaryMic 

![Untitled](https://github.com/2shady4u/godot-sqlite/assets/42484461/b4373f5d-6fae-430d-93a7-791bfd8c032f)

I just checked this on Windows using Godot 4.2.1 and Godot-SQLite v4.3. As you can see this feature is already implemented 🤔 

Could you tell me:
- What OS are you using? (Windows? Linux? ....)
- What version of the Godot-SQLite plugin are you using?

**Comment by StaryMic** [comment]
So weirdly enough. This issue is suddenly fixed for me. I opened the main project where I was gonna use this plugin and installed it there and now I suddenly have the autocomplete results?? I just installed it under a different project and enabled the plugin just like I already did and now it works. 
![image](https://github.com/2shady4u/godot-sqlite/assets/37278662/aef3d1a0-b705-4643-8773-d6a290c5b2d2)

I'm on Windows 10 using version 4.3 of Godot-SQLite.

I've got no clue why it works now, but thank you for taking a look into it!
